### DISCUSSION TITLE
Don't open Repository link in new tab by default

### DISCUSSION THREAD

**Discussion by stephan281094** [community/community#128416] [npm]
![Screenshot 2023-11-09 at 12 59 06](https://github.com/npm/feedback/assets/4006802/91dee9ac-2315-4e14-b284-e34ca927f6db)

Hi there! I often use `https://npmjs.org/package/{package-name}` to find the repository where the code of a package is located. The npm website conveniently displays a "Repository" link if one has been specified inside of `package.json`. Right now, when clicking this link, the npm website forces the link to be opened in a new tab. In 99% of the cases when I click this link, however, I want it to open in the same tab. It's a frustrating experience to have to close the npm tab every time I just want to go to the repository.

In my opinion, the user should be in charge. They should be able to decide whether to open the link in a new tab or not. This is also inconsistent with the "Homepage" link, where the link is _not_ automatically opened in a new tab.

Thank you for taking the time to read this and I'm curious to hear your thoughts!

**Comment by panva** [comment]
> This is also inconsistent with the "Homepage" link, where the link is not automatically opened in a new tab.

Consistency should be there. But I for instance, in 99% of cases, prefer a new tab :)

**Reply by stephan281094** [reply]
I think that’s fine, because you could still do so with cmd/ctrl + click. You’d have the option to opt in, whereas right now there’s no way to opt out of this behavior.
